ambry-api/src/main/java/com.github.ambry/router/GetBlobResult.java

```diff
@@ -32,6 +35,13 @@
   public GetBlobResult(BlobInfo blobInfo, ReadableStreamChannel blobDataChannel) {
     this.blobInfo = blobInfo;
     this.blobDataChannel = blobDataChannel;
+    this.blobChunkIds = null;
```

Comment:
I would prefer just removing the 2 arg constructor up here and have GetBlobInfoOperation pass in null here.

The intellij "refactor->change signature" tool helps with this.